CHANGELOG_iOS.md

```diff
@@ -1,22 +1,58 @@
 # Changelog
 
+## [1.68.134](https://github.com/brave/brave-browser/releases/tag/v1.68.134)
+
+### Web3
+
+ - Added the ability to import BTC accounts (BIP84) into Brave Wallet. ([#38546](https://github.com/brave/brave-browser/issues/38546))
+ - Added the ability to send BTC on “Send” screen. ([#37958](https://github.com/brave/brave-browser/issues/37958))
+ - Updated default account names to differentiate between mainnet and testnet accounts. ([#38526](https://github.com/brave/brave-browser/issues/38526))
+ - Updated “Show Zero Balances” button to always be displayed in “Select a Token to Send” screen. ([#38452](https://github.com/brave/brave-browser/issues/38452))
```

Comment:
Maybe change to 
`Updated “Show Zero Balances” button to be always shown on "Select a Token to Send” screen.`